docs/UserGuide.md

```diff
@@ -247,37 +247,37 @@ You can use this command to see the entire list of students and classes.
 For example:
 1. After finding the class CS2103T and student Alex, you wish to view all your students and classes.
 
-1. You can type in the command `list` and press <kbd>Enter</kbd>.
+1. You type in the command `list` and press <kbd>Enter</kbd>.
 
    ![List before](images/ugimages/ListBefore.png)
 
-1. Now you can see all your students and classes.
+1. Now, you can see all your students and classes.
 
    ![List after](images/ugimages/ListAfter.png)
 
 #### Undoing previous changes : `undo`
 
-If you have accidentally entered a command that permanently changes the data in your Tutor's Pet,
-you can use this command to undo the changes.
+If you have accidentally entered a command that permanently modifies the data in your Tutor's Pet,
+you can use this command to undo the modification.
```

Comment:
I would be inclined to retain the term `change` instead since it has been used in multiple other places like view change history etc.